bedrock/settings/base.py

```diff
@@ -952,7 +955,7 @@ def lazy_email_backend():
 }
 
 MOFO_SECURITY_ADVISORIES_PATH = config('MOFO_SECURITY_ADVISORIES_PATH',
-                                       default=path('..', 'mofo_security_advisories'))
+                                       default=path('mofo_security_advisories'))
```

Comment:
I got a little cache happy. This helps me speed up demo deploys by making it easier to cache the MFSA repo, then include it in the docker build, then running the command in the cron process in the demos doesn't have to clone the whole repo every deploy. I can split it out though if you'd prefer.